main.py

```diff
@@ -26,59 +31,79 @@ def validate_number(num):
         return False
 
 
-def validate_operator(operator):
+def validate_operator(operator: any) -> bool:
+    """Validate value for being in a list of operators and print a massage if the validation failed."""
     if operator in valid_operators:
         return True
 
     print(f'Please enter one on these operators: {", ".join(valid_operators)}')
     return False
 
 
-def handle_input(phrase, validator):
+def handle_input(phrase: str, validator: Callable[[any], bool]) -> T:
```

Comment:
according to previous comments a small modification here: validator: Callable[[str], bool]